db-service/lib/cqn2sql.js

```diff
@@ -874,7 +874,6 @@ class CQN2SQLRenderer {
       if (converter && sql[0] !== '$') sql = converter(sql, element)
 
       let val = _managed[element[annotation]?.['=']]
-        || !isUpdate && (element['@cds.valid.from'] ? '$valid.from' : element['@cds.valid.to'] ? '$valid.to' : null)
```

Comment:
This does not look correct. If the session context contains the `from` and `to` it should be considered.